pkgs/applications/version-management/bit/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "bit";
+  version = "0.5.11";
+
+  src = fetchFromGitHub {
+    owner = "chriswalz";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rdi6wsr5w64gvqkdysxrxfjxi3xbwigy3sk8wsx9zp4fql9xnw1";
+  };
+
+  subPackages = [ "." ];
+
+  vendorSha256 = "16c1mcvz3z17pixp904hazivx2n7y592nlx3ndnyajk7ljmnwfzy";
+
+  meta = with lib; {
```

Comment:
Should be wrapped with `git` in `$PATH`, per: https://github.com/chriswalz/bit#using-go-harder-way-to-install

Reply:
Just adding to buildInputs should suffice?

Reply:
No. that won't affect it at all. Use something like this:

https://github.com/NixOS/nixpkgs/blob/62e614262ad4b9df3d40693350f516794e95ecda/pkgs/tools/security/gopass/default.nix#L49-L55